gis/gdal/Portfile

```diff
@@ -67,214 +67,202 @@ depends_lib-append  port:zlib \
+                    -DGDAL_ENABLE_DRIVER_GFF=ON              \
+                    -DGDAL_ENABLE_DRIVER_GRIB=ON             \
+                    -DGDAL_ENABLE_DRIVER_GSG=ON              \
+                    -DGDAL_ENABLE_DRIVER_GXF=ON              \
+                    -DGDAL_ENABLE_DRIVER_HF2=ON              \
+                    -DGDAL_ENABLE_DRIVER_ILWIS=ON            \
+                    -DGDAL_ENABLE_DRIVER_IRIS=ON             \
+                    -DGDAL_ENABLE_DRIVER_JAXAPALSAR=ON       \
+                    -DGDAL_ENABLE_DRIVER_JDEM=ON             \
+                    -DGDAL_ENABLE_DRIVER_L1B=ON              \
+                    -DGDAL_ENABLE_DRIVER_LEVELLER=ON         \
+                    -DGDAL_ENABLE_DRIVER_MAP=ON              \
+                    -DGDAL_ENABLE_DRIVER_MRF=ON              \
+                    -DGDAL_ENABLE_DRIVER_MSGN=ON             \
+                    -DGDAL_ENABLE_DRIVER_NGSGEOID=ON         \
+                    -DGDAL_ENABLE_DRIVER_NITF=ON             \
+                    -DGDAL_ENABLE_DRIVER_NOAA_B=ON           \
+                    -DGDAL_ENABLE_DRIVER_NORTHWOOD=ON        \
+                    -DGDAL_ENABLE_DRIVER_NSIDCBIN=ON         \
+                    -DGDAL_ENABLE_DRIVER_OPENFILEGDB=ON      \
+                    -DGDAL_ENABLE_DRIVER_OZI=ON              \
+                    -DGDAL_ENABLE_DRIVER_PCIDSK=ON           \
+                    -DGDAL_ENABLE_DRIVER_PDS=ON              \
+                    -DGDAL_ENABLE_DRIVER_PNG=ON              \
+                    -DGDAL_ENABLE_DRIVER_PRF=ON              \
+                    -DGDAL_ENABLE_DRIVER_R=ON                \
+                    -DGDAL_ENABLE_DRIVER_RAW=ON              \
+                    -DGDAL_ENABLE_DRIVER_RMF=ON              \
+                    -DGDAL_ENABLE_DRIVER_RS2=ON              \
+                    -DGDAL_ENABLE_DRIVER_SAFE=ON             \
+                    -DGDAL_ENABLE_DRIVER_SAGA=ON             \
+                    -DGDAL_ENABLE_DRIVER_SAR_CEOS=ON         \
+                    -DGDAL_ENABLE_DRIVER_SDTS=ON             \
+                    -DGDAL_ENABLE_DRIVER_SENTINEL2=ON        \
+                    -DGDAL_ENABLE_DRIVER_SGI=ON              \
+                    -DGDAL_ENABLE_DRIVER_SIGDEM=ON           \
+                    -DGDAL_ENABLE_DRIVER_SRTMHGT=ON          \
+                    -DGDAL_ENABLE_DRIVER_STACIT=ON           \
+                    -DGDAL_ENABLE_DRIVER_STACTA=ON           \
+                    -DGDAL_ENABLE_DRIVER_TERRAGEN=ON         \
+                    -DGDAL_ENABLE_DRIVER_TGA=ON              \
+                    -DGDAL_ENABLE_DRIVER_TIL=ON              \
+                    -DGDAL_ENABLE_DRIVER_TSX=ON              \
+                    -DGDAL_ENABLE_DRIVER_USGSDEM=ON          \
+                    -DGDAL_ENABLE_DRIVER_XPM=ON              \
+                    -DGDAL_ENABLE_DRIVER_XYZ=ON              \
+                    -DGDAL_ENABLE_DRIVER_ZMAP=ON
```

Comment:
Are you sure all these flags are correct? The CI logs say:
```
CMake Warning:
    Manually-specified variables were not used by the project:
  
      CMAKE_OBJCXX_COMPILER
      CMAKE_OBJC_COMPILER
      CMAKE_POLICY_DEFAULT_CMP0025
      CMAKE_POLICY_DEFAULT_CMP0060
      GDAL_ENABLE_DRIVER_AIG
      GDAL_ENABLE_DRIVER_ESAT
      GDAL_ENABLE_DRIVER_NOAA_B
      GDAL_ENABLE_DRIVER_NSIDCBIN
      GDAL_ENABLE_DRIVER_OPENFILEGDB
      GDAL_ENABLE_DRIVER_WFS
      GDAL_USE_CHARLS
      GDAL_USE_CRYPTO
      GDAL_USE_EXR
      GDAL_USE_FME
      GDAL_USE_JASPER
      GDAL_USE_JP2MRSID
      GDAL_USE_KAKADU
      GDAL_USE_LIBZ
      GDAL_USE_MONGOCXXV3
      GDAL_USE_MSG
      GDAL_USE_OCI
      GDAL_USE_PAM
      GDAL_USE_PERL
      GDAL_USE_PG
      GDAL_USE_PYTHON
      GDAL_USE_XERCES
      GDAL_USE_XML2
      OGR_ENABLE_DRIVER_AVCBIN
      OGR_ENABLE_DRIVER_AVCE00
      OGR_ENABLE_DRIVER_GTFS
      OGR_ENABLE_DRIVER_ZARR
```

Reply:
> Are you sure all these flags are correct? The CI logs say:
> 
> ```
> CMake Warning:
>     Manually-specified variables were not used by the project:
>   
>       CMAKE_OBJCXX_COMPILER
> ...
> ```

These were all addressed, either removed or renamed.